foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/credentials/internal/CredentialMessages.properties

```diff
@@ -18,3 +18,14 @@ SelectDomain=Please select a domain.
 UsernameCannotBeBlank=Username cannot be blank.
 PasswordCannotBeBlank=Password cannot be blank.
 UsernameAlreadyExists=Username {0} already exists for domain {1}.
+
+# For use in the credential prompter
+OK=OK
+Cancel=Cancel
+CredentialPrompterTitle=Password Required
```

Comment:
https://wiki.eclipse.org/Talk:Capitalization#Capitalization

Capitalization

Close attention should be paid to capitalization within dialogs and windows. Headline capitalization should be applied to all titles, menus, tooltips, tabs and push buttons. Sentence style capitalization should be applied to all check boxes, radio buttons and group labels. 

---

So it seems headline capitalization as declared by Associated Press (and others) is appropriate for title bars.